pairing/bls12381/suite_test.go

```diff
@@ -0,0 +1,365 @@
+package bls12381
+
+import (
+	"bytes"
+	"crypto/cipher"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"go.dedis.ch/kyber/v3"
+	"go.dedis.ch/kyber/v3/util/random"
+)
+
+// Code extracted from kyber/utils/test
+// TODO: expose API in forked drand/kyber
+// Apply a generic set of validation tests to a cryptographic Group,
+// using a given source of [pseudo-]randomness.
+//
+// Returns a log of the pseudorandom Points produced in the test,
+// for comparison across alternative implementations
+// that are supposed to be equivalent.
+func testGroup(t *testing.T, g kyber.Group, rand cipher.Stream) []kyber.Point {
```

Comment:
Long time I didn't look at the code, so I might be missing something obvious: why did you copy the test here, instead of adding the new suite to the list of suites to be tested?

Reply:
Can you point me to where the list of suites to be tested? It's my first time submitting code to this repo. I might miss something obvious as well.

Reply:
OK, got it wrong: in fact the suites themselves call the tests:

- group/edwards25519/curve_test.go
- group/nist/group_test.go

But I'm not sure it works with pairing-based suites, as some of the methods of the `Point` and `Scalar` interface might not be defined.

So what I'm proposing is something like this: instead of copying the lines from the `util/test/test.go` file, do something like:

```
func TestG1(t *testing.T) {
	test.GroupTest(t, newGroupG1())
}
```

Reply:
I tries and that works. So yes, please use the functions from `util/test/`.